save-backend/src/main/kotlin/com/saveourtool/save/backend/service/OrganizationService.kt

```diff
@@ -101,6 +101,20 @@ class OrganizationService(
     fun getByName(name: String) = findByName(name)
         ?: throw NoSuchElementException("There is no organization with name $name.")
 
+    /**
+     * @param prefix prefix of organization name
+     * @param status status of required organizations
+     * @return list of organizations with required [status] which names start with [prefix]
+     */
+    fun getByPrefixAndStatus(prefix: String, status: OrganizationStatus) = if (prefix.isBlank()) {
```

Comment:
Let's delete the getNotDeletedOrganizations function now. And we use your function. This is the same function as this one, but with hardcoded stats.

Reply:
Done